tests/io_perfomance/image/config.fio

```diff
@@ -0,0 +1,183 @@
+[global]
```

Comment:
Do we really need 2 configs which look the same. 
And two images?

Reply:
Same question here -- and see the question about README above.